nservicebus/errors/automatic-retries.md

```diff
@@ -31,7 +31,7 @@ Note: The configured value describes the minimum number of times a message will
 
 ### Transport transaction requirements
 
-The FLR mechanism is implemented by rolling back the [transport transaction](/nservicebus/messaging/transactions.md). This returns the message to the input queue and the endpoint processes it again immediately. Therefore FLR cannot be used when transport transactions are disabled. In such cases aborting the receive operation results in message loss.
+The FLR mechanism is implemented by rolling back the [transport transaction](/nservicebus/messaging/transactions.md). This returns the message to the input queue and the endpoint processes it again immediately. Aborting the receive operation when transactions are turned off would results in message loss. Therefore FLR cannot be used when transport transactions are disabled. 
```

Comment:
Meaning that even if they're configured by the user they won't be used or that if you try to enable them then exception is thrown? Might be good to clarify what happens under the hoods.